.NET/Microsoft.Recognizers.Definitions.Common/Chinese/DateTimeDefinitions.cs

```diff
@@ -55,11 +55,11 @@ public static class DateTimeDefinitions
       public static readonly string DateRegexList1 = $@"({LunarRegex}(\s*))?((({SimpleYearRegex}|{DateYearInChineseRegex})年)(\s*))?{MonthRegex}(\s*){DateDayRegexInChinese}((\s*|,|，){WeekDayRegex})?";
       public static readonly string DateRegexList2 = $@"((({SimpleYearRegex}|{DateYearInChineseRegex})年)(\s*))?({LunarRegex}(\s*))?{MonthRegex}(\s*){DateDayRegexInChinese}((\s*|,|，){WeekDayRegex})?";
       public static readonly string DateRegexList3 = $@"((({SimpleYearRegex}|{DateYearInChineseRegex})年)(\s*))?({LunarRegex}(\s*))?{MonthRegex}(\s*)({DayRegexNumInChinese}|{DayRegex})((\s*|,|，){WeekDayRegex})?";
-      public static readonly string DateRegexList4 = $@"{MonthNumRegex}\s*/\s*{DayRegex}((\s+|\s*,\s*){SimpleYearRegex})?";

```

Comment:
Had that commit that caused the side-effect also changed these? I thought it only changed regexes 5, 6, and 7. But maybe I misremember.

Reply:
Because the order of month and day in X/X(M/D) is different from X/X/Y(D/M/Y), I changed DateRegexList4/5 to only support X/X pattern. Separately extract X/X and X/X/Y pattern.